### ISSUE TITLE
Aggregated Cost Model API window parameter behavior does not match documentation

### ISSUE THREAD

**Issue by joerocklin** [kubecost/cost-model#226]
The documentation for the `window` parameter on the [aggregated cost model API](https://github.com/kubecost/docs/blob/master/allocation-api.md#aggregated-cost-model-api) states: 

> `window` dictates the applicable window for measuring cost metrics. Supported units are d, h, and m.

My original testing was using `86400s` and it was working just fine. Then I tried with 300s and received the following error:

```json
{
    "code": 500,
    "status": "error",
    "data": null,
    "message": "invalid parameter 'query': parse error at line 2, char 37: duration must be greater than 0"
}
```

Some testing uncovered that a value of `3600s` works, but `3599s` does not. Furthermore, a value of `60m` works, but `59m` also returns the above error. Some more playing with including or excluding various fields showed that the above error only occurs with windows smaller than 1 hour if the `allocateIdle` flag is set to `true`, otherwise the call returns as expected.

It seems like:
1. The documentation should be updated to reflect that `s` is also a valid time unit
2. The documentation should be updated to reflect that if `allocateIdle` is set to `true`, the smallest unit of time valid for the `window` field is 1 hour's worth of units.
3. The error message returned to the user if `allocateIdle` is true and the time window is less than an hour should reflect that the duration provided is greater than one hour instead of the message indicating it is greater than zero.

Or, perhaps the interplay between `allocateIdle` and the window duration is actually a bug in its own right.

**Comment by dwbrown2** [comment]
Thanks so much for the thorough report, Joe! I'm able to recreate with the following message:

`Error parsing normalization values: end timestamp must not be before start time`

We'll investigate what's causing this and get docs, messaging, etc. updated

**Comment by joerocklin** [comment]
@dwbrown2 After upgrading to 1.47.0 I'm also getting the error you're showing. But this time it is with just the params `?aggregation=namespace&window=59m` defined. It seems as though the minimum time window for any query with 1.47.0 is 1 hour.

**Comment by dwbrown2** [comment]
Hi Joe, we have officially added seconds as a supported windowing option. Docs are now updated to reflect this. In v1.47.1 (released last night), we added support for queries with windows < 60m when `allocateIdle=false`. We'll be making this window size available for all API request options in our upcoming build. Will update this bug once it's available. 

Let me know if I can get you anything else here.
